test/integration/features/cmd/version.feature

```diff
@@ -0,0 +1,7 @@
+@minishift-only @command
+Feature: Version command
+Command "minishift version" shows version of minishift binary.
+
+  Scenario: Setup-cdk installs and enables specific addons
```

Comment:
@hferentschik Yes, thanks for pointing out. So small file and even though I missed that...